pkg/controller/update_cluster.go

```diff
@@ -556,25 +556,36 @@ func (c *M3DBController) shrinkPlacementForSet(
 		return err
 	}
 
-	_, removeInst, err := c.findPodInstanceToRemove(cluster, pl, pods)
+	_, removeInst, err := c.findPodInstancesToRemove(cluster, pl, pods, removeCount)
 	if err != nil {
 		c.logger.Error("error finding pod to remove", zap.Error(err))
 		return err
 	}
 
-	c.logger.Info("removing pod from placement", zap.String("instance", removeInst.ID()))
-	return c.adminClient.placementClientForCluster(cluster).Remove([]string{removeInst.ID()})
+	if len(removeInst) == 0 {
+		c.logger.Info("nothing to remove, skipping remove call")
+		return nil
+	}
+
+	removeIds := make([]string, len(removeInst))
+	for idx, inst := range removeInst {
+		removeIds[idx] = inst.ID()
+	}
+	c.logger.Info("removing instances from placement",
+		zap.String("instances", strings.Join(removeIds, ",")))
+	return c.adminClient.placementClientForCluster(cluster).Remove(removeIds)
 }
 
-// findPodInstanceToRemove returns the pod (and associated placement instace)
+// findPodInstancesToRemove returns the pod (and associated placement instace)
```

Comment:
Nit: can you update the comment to reflect that this will return multiple pods / instances now?